test/libyul/ObjectCompilerTest.cpp

```diff
@@ -57,7 +57,7 @@ ObjectCompilerTest::ObjectCompilerTest(string const& _filename):
 			{"standard", OptimisationPreset::Standard},
 			{"full", OptimisationPreset::Full},
 		},
-		"minimal"
+		"none"
```

Comment:
I'm switching the default to `none` in `ObjectCompiler` tests, because `minimal` now optimizes out stuff that used to stay with optimizer disabled. I can make it `minimal` if you think it's better but I'm not sure if I'll be able to fix all test cases then.

Reply:
No longer relevant. Now Yul optimizer cannot be disabled even via `OptimiserSettings`, so I have no choice but to do it like this.